assets/src/edit-story/elements/text/edit.js

```diff
@@ -60,6 +60,15 @@ import {
 const Wrapper = styled.div`
   ${elementFillContent}
   ${elementWithBackgroundColor}
+
+  span[data-text="true"] {
```

Comment:
This depends on a feature of `draftjs`, right? So if they happened to change this annotation, this would break?

Reply:
Oh no, I completely broke this. Try having a text field with padding and mixed inline styling (some bold, some non-bold) - each separately styled sequence of characters will be wrapped in its own `<span data-text="true" />` and the sentence will thus have inline padding in-between characters:

![broken-text-padding](https://user-images.githubusercontent.com/637548/94312619-3977bd00-ff4b-11ea-893f-2a08475f1759.gif)

Reply:
Yes, that's the downside, not sure if there is a clean way to future proof it (maybe we could add a class to every span with text contents in DraftJS and then target it here?).

Reply:
@barklund That complicates things, I will look into it, but the worst part is that `box-decoration-break: clone;` doesn't work for multiple span elements (combining them into one).